packages/feedback/src/FeedbackForm.js

```diff
@@ -218,6 +219,7 @@ FeedbackForm.propTypes = {
       value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     })
   ),
+  customPlaceholder: PropTypes.string,
```

Comment:
this prop type should be `PropTypes.node` per the react select docs https://react-select.com/props